@stellar/typescript-wallet-sdk/test/wallet.test.ts

```diff
@@ -1880,3 +1884,56 @@ describe("Http client", () => {
     expect(resp.data.transaction).toBeTruthy();
   });
 });
+
+describe("AuthHeaderSigner", () => {
```

Comment:
IMO a good idea would be to test that this token works with anchor platform, but it's not merged yet (maybe once it's done)?
https://github.com/stellar/java-stellar-anchor-sdk/pull/1303

Reply:
ya agreed, I'll add a small ticket to track

https://stellarorg.atlassian.net/browse/WAL-1441